packages/lodestar-utils/test/unit/logger/format.test.ts

```diff
@@ -1,21 +1,25 @@
 import {serializeContext} from "../../../src/logger";
 import {expect} from "chai";
-import {config} from "@chainsafe/lodestar-config/lib/presets/minimal";
 
 describe("log format", function () {
 
   describe("serialize context", function () {
     it("should serialize context", function () {
       const serialized = serializeContext({
-        array: config.types.Uint64.hashTreeRoot(10n),
-        buffer: Buffer.alloc(2, 0),
+        object: {
+          foo: "bar",
+          baz: 1,
+        },
+        array: ["a", "small", "array"],
         string: "test",
-        bigint: 10n
+        number: 3,
+        bool: true,
       });
       expect(serialized)
         .to.equal(
-          "array=0x0a00000000000000000000000000000000000000000000000000000000000000,"
```

Comment:
I really liked auto serializing arrays to hex since we have a lot of those throughout the code

Reply:
There was actually only one instance of using this feature, all else already are either `toJson`/`toHexString`ed.